cmd/heznode/cfg.buidler.toml

```diff
@@ -7,32 +7,39 @@ MaxSQLConnections = 100
 SQLConnectionTimeout = "2s"
 
 [PriceUpdater]
-Interval = "10s"
-URLBitfinexV2 = "https://api-pub.bitfinex.com/v2/"
-URLCoinGeckoV3 = "https://api.coingecko.com/api/v3/"
+Interval = "5s"
+Priority = "bitfinexV2,CoinGeckoV3"
+Statictokens="3=1.31,4=1.01"
+
+[PriceUpdater.Fiat]
+APIKey=""
+URL="https://api.exchangeratesapi.io/v1/"
+BaseCurrency="USD"
+Currencies="CNY,EUR,JPY,GBP"
+
+[[PriceUpdater.Provider]]
+Provider = "bitfinexV2"
+BASEURL = "https://api-pub.bitfinex.com/v2/"
+URL = "ticker/t"
+URLExtraParams = "USD"
+# token order ETH,HEZ,USDT,USDC,DAI,WBTC,WETH,XAUt,UNI,SUSHI,COMP,BAL,AAVE,YFI,LINK. Order based in tokenId.
 # Available update methods:
-# - coingeckoV3 (recommended): get price by SC addr using coingecko API
-# - bitfinexV2: get price by token symbol using bitfinex API
-# - static (recommended for blacklisting tokens): use the given StaticValue to set the price (if not provided 0 will be used)
-# - ignore: don't update the price leave it as it is on the DB
-DefaultUpdateMethod = "coingeckoV3" # Update method used for all the tokens registered on the network, and not listed in [[PriceUpdater.TokensConfig]]
-[[PriceUpdater.TokensConfig]]
-UpdateMethod = "bitfinexV2"
-Symbol = "USDT"
-Addr = "0xdac17f958d2ee523a2206206994597c13d831ec7"
-[[PriceUpdater.TokensConfig]]
-UpdateMethod = "coingeckoV3"
-Symbol = "ETH"
-Addr = "0x0000000000000000000000000000000000000000"
-[[PriceUpdater.TokensConfig]]
-UpdateMethod = "static"
-Symbol = "UNI"
-Addr = "0x1f9840a85d5af5bf1d1762f925bdaddc4201f984"
-StaticValue = 30.12
-[[PriceUpdater.TokensConfig]]
-UpdateMethod = "ignore"
-Symbol = "SUSHI"
-Addr = "0x6b3595068778dd592e39a122f4f5a5cf09c90fe2"
+# - ignore method don't update the price leave it as it is on the DB. Type ignore or leave the field empty
+# - static method is performed writting the price in Statictokens.
+# Example: Symbols = "0=ETH,1=HEZ,2=ignore,4=DAI,5=WBT,6=static,7=XAUT:,8=UNI,9=SUSHI:,10=COMP:,11=,12=AAVE:,13=YFI,14=LINK:"
+Symbols = "0=ETH,2=UST,8=ignore,9=SUSHI:,5=WBT,14=LINK:,12=AAVE:,7=XAUT:,10=COMP:,6=ETH"
+
+[[PriceUpdater.Provider]]
+Provider = "CoinGeckoV3"
+BASEURL = "https://api.coingecko.com/api/v3/"
+URL = "simple/token_price/ethereum?contract_addresses="
+URLExtraParams = "&vs_currencies=usd"
+# token order ETH,HEZ,USDT,USDC,DAI,WBTC,WETH,XAUt,UNI,SUSHI,COMP,BAL,AAVE,YFI,LINK. Order based in tokenId.
+# Available update methods:
+# - ignore method don't update the price leave it as it is on the DB. Type ignore or leave the field empty
+# - static method is performed writting the price in Statictokens.
+# Example: Addresses="0=0x00,1=0x12,2=0x12,4=0x12,5=0x12,6=,7=0x12,8=,9=0x12,10=0x12,11=ignore,12=0x12,13=0x12,14=0x12"
+Addresses="0=0x0000000000000000000000000000000000000000,2=0xdac17f958d2ee523a2206206994597c13d831ec7,8=ignore"
```

Comment:
I don't understand the neeed for tis, we already know the addr of the tokens thanks to the synchronizer

Reply:
In the same way as we were doing it before, this allows to force and use a specific token address. If the token address is not specified, then the service tries to use the address stored in token table as you has mentioned